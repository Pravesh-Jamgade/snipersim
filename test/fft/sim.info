{'cmdline': ['../../run-sniper',
             '-v',
             '-n',
             '1',
             '-c',
             'gainestown',
             '--',
             './fft',
             '-p',
             '1'],
 'git_revision': '527b99e06f8fc217cf4b2616a7151f0114c7c915',
 'host': 'ubuntu-PowerEdge-R740',
 'pin_version': 'unknown',
 'rusage': (2.060116,
            0.10585499999999999,
            41124,
            0,
            0,
            0,
            15882,
            0,
            0,
            0,
            144,
            0,
            0,
            0,
            10059,
            5),
 'snipercmd': ['bash',
               '-c',
               '/media/ubuntu/Data/saurabh/Simulator/sniper/lib/sniper -c /media/ubuntu/Data/saurabh/Simulator/sniper/config/base.cfg --general/total_cores=1 --general/output_dir=/media/ubuntu/Data/saurabh/Simulator/sniper/test/fft --config=/media/ubuntu/Data/saurabh/Simulator/sniper/config/nehalem.cfg --config=/media/ubuntu/Data/saurabh/Simulator/sniper/config/gainestown.cfg -g --traceinput/stop_with_first_app=true -g --traceinput/restart_apps=false -g --traceinput/stop_with_first_app=false -g --traceinput/enabled=true -g --traceinput/emulate_syscalls=true -g --traceinput/num_apps=1 -g --traceinput/trace_prefix=/tmp/tmpun1X20/run_benchmarks'],
 't_elapsed': 5.889527082443237,
 't_start': 1682407768.063664,
 'tracecmds': [['/media/ubuntu/Data/saurabh/Simulator/sniper/record-trace',
                '-o',
                '/tmp/tmpun1X20/run_benchmarks',
                '-v',
                '-e',
                '1',
                '-s',
                '0',
                '-r',
                '1',
                '--follow',
                '--routine-tracing',
                '--',
                './fft',
                '-p',
                '1']],
 'user': 'saurabh',
 'vmem': 43061248L}
