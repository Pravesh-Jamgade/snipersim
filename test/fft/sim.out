                                     | Core 0    
  Instructions                       |    1614895
  Cycles                             |    1348354
  IPC                                |       1.20
  Time (ns)                          |     506900
  Idle time (ns)                     |        -87
  Idle time (%)                      |      -0.0%
Branch predictor stats               |           
  num correct                        |      71535
  num incorrect                      |       5219
  misprediction rate                 |      6.80%
  mpki                               |       3.23
TLB Summary                          |           
  I-TLB                              |           
    num accesses                     |     165075
    num misses                       |         80
    miss rate                        |      0.05%
    mpki                             |       0.05
  D-TLB                              |           
    num accesses                     |     853459
    num misses                       |         58
    miss rate                        |      0.01%
    mpki                             |       0.04
  L2 TLB                             |           
    num accesses                     |        138
    num misses                       |        136
    miss rate                        |     98.55%
    mpki                             |       0.08
Cache Summary                        |           
  Cache L1-I                         |           
    num cache accesses               |     165075
    num cache misses                 |       1173
    miss rate                        |      0.71%
    mpki                             |       0.73
  Cache L1-D                         |           
    num cache accesses               |     853459
    num cache misses                 |       3019
    miss rate                        |      0.35%
    mpki                             |       1.87
  Cache L2                           |           
    num cache accesses               |       4324
    num cache misses                 |       2416
    miss rate                        |     55.87%
    mpki                             |       1.50
  Cache L3                           |           
    num cache accesses               |       2703
    num cache misses                 |       2416
    miss rate                        |     89.38%
    mpki                             |       1.50
DRAM summary                         |           
  num dram accesses                  |       2414
  average dram access latency (ns)   |      71.03
  average dram queueing delay        |      17.60
  average dram bandwidth utilization |      4.00%
Coherency Traffic                    |           
  num loads from dram                |        547
  num loads from dram cache          |          0
  num loads from remote cache        |          0
